### ISSUE TITLE
Update claim status tool FAQ with download decision letter feature

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#56984]
## Issue Description
We need to update the [claim status tool FAQ page](https://www.va.gov/resources/claim-status-tool-faqs/) to reflect the new feature that lets Veterans download decision letters. 

We'll need to adjust these 2 Q&As:
- What kind of information can I get in this tool?
- Can the tool tell me when I’ll receive a decision letter for my claim?

---
## Tasks
- [x] Draft updates (Katherine)
- [x] Review (Laura)
- [ ] Copyedit (Megan)
- [ ] Share with partners and product team
- [ ] Prepub and publish (Megan)

## Acceptance Criteria
- [ ] FAQ reflects new tool feature

**Comment by katherine-fung** [comment]
Note to self: 

How can I find out the status of my claim, decision review, or appeal? 
http://preview-prod.vfs.va.gov/preview?nodeId=8504

What kind of information can I get in this tool? 
http://preview-prod.vfs.va.gov/preview?nodeId=8515
[changed the order of "claim, decision review, or appeal";
added bullet for "Decision letters about your claim, decision review, or appeal";
tweaked the note]

Can the tool tell me when I’ll receive a decision letter for my claim?
http://preview-prod.vfs.va.gov/preview?nodeId=8521

[existing copy: 
The claim status tool won’t provide an exact date. But if the status of your claim is “complete,” you can estimate when you’ll receive your decision letter. This status means we’ve already mailed your letter, and it will take 7 to 10 business days to arrive from the date we mailed it.

Note: The tool will show this message just below the status on the claim: “We sent you a decision letter.”

If you don’t receive your letter within 10 days, please call us at 800-827-1000 (TTY: 711). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET. ]

**Comment by katherine-fung** [comment]
Hi @laurwill, 

The multiple FAQ page - and the single FAQs within it - are ready for review:

**How can I find out the status of my claim, decision review, or appeal?** 
http://preview-prod.vfs.va.gov/preview?nodeId=8504

- I changed the order of "claim, decision review, or appeal" in this single FAQ. 

**What kind of information can I get in this tool?** 
http://preview-prod.vfs.va.gov/preview?nodeId=8515

- I changed the order of "claim, decision review, or appeal" in this single FAQ. 
- I added a line and 2 bullets to say you can get decision letters for these types of claims, decision reviews, or appeals. 
- I tweaked the note about the tool not showing documents that contain protected information. It still sounds a bit weird. Should we get clarity on what this means? I wasn't sure if it was referring to protected health information, or other personal information. 

**Can the tool tell me when I’ll receive a decision letter for my claim?**
http://preview-prod.vfs.va.gov/preview?nodeId=8521

- I added information about being able to download your decision letter for certain types of claims, decision reviews, and appeals. 
- I kept the live copy that's still relevant for other types of claims. I changed "if the status is complete" to "if the status is closed" to reflect the language in the tool. For this section, I also removed the note, "
Note: The tool will show this message just below the status on the claim: 'We sent you a decision letter.'" I don't think it's necessary. 

**entire page**
http://preview-prod.vfs.va.gov/preview?nodeId=9275

**Comment by laurwill** [comment]
Thanks @katherine-fung ! I made a few edits:
- Kept the types of claims, etc, that allow downloading letters more general so that we don't need to keep updating this page every time they add a new type. See what you think of that change.
- Adjusted the Q&A about estimated time to get the letter to lead with the mailed option and then mention downloads. This seemed simpler to me, but lmk if you disagree. 

As we chatted, the "closed" status in the tool is not consistent with our page about claim statuses. So we'll need to confirm the status terms before we publish this change. Stay tuned!
